src/theme.config

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
Just realized that you aren't committing the auto generated css file `semantic.min.css` since react is reading from this same repo you have to commit them too. And check how your changes affect the react app.

Reply:
@shakirandagire this file isn't in the .gitignore file

Reply:
I have committed the `semantic.min.css`